examples/react/src/components/Content.tsx

```diff
@@ -7,6 +13,8 @@ import { useWalletSelector } from "../contexts/WalletSelectorContext";
 import SignIn from "./SignIn";
 import Form from "./Form";
 import Messages from "./Messages";
+import { Components } from "@near-wallet-selector/selector-ui/loader";
+import { WalletSelectorModal } from "@near-wallet-selector/react-selector-ui";
```

Comment:
I'm actually a little confused about these dependencies.

When I look at [Iconic](https://ionicframework.com/react) (the main user of Stencil it seems), they have no concept of this "loader" module.

I'm also a little skeptical on whether we _actually_ need the framework specific packages if we just used Web Components directly which is supported by these frameworks and "native" Web? The reason I have this concern is it's starting to feel like the "simple modal" package installation flow is a lot more work than it should be from the perspective of a developer building a new dApp

Reply:
I will investigate on this one since I am not sure about this myself.